node/src/components/chain_synchronizer/operations.rs

```diff
@@ -633,18 +643,34 @@ async fn sync_deploys_and_transfers_and_state(
         .cloned()
         .collect();
     let mut stream = futures::stream::iter(hash_iter)
-        .map(|hash| fetch_and_store_deploy(effect_builder, hash))
+        .map(|hash| {
+            debug!("start - fetch_and_store_deploy - archival sync - {}", hash);
```

Comment:
This is reversed, isn't it? Other logs log start at INFO and finish at DEBUG and here it's the opposite.

Reply:
Here both "start" and "finish" was intended as `debug` from the start, as this is expected to be called very often. See the PR description.